src/core/utils/matching/normalizePath.ts

```diff
@@ -6,16 +6,15 @@ import { getAbsoluteUrl } from '../url/getAbsoluteUrl'
  * Normalizes a given request handler path:
  * - Preserves RegExp.
  * - Removes query parameters and hashes.
- * - Rebases relative URLs against the "baseUrl" or the current location.
  * - Preserves relative URLs in Node.js, unless specified otherwise.
  */
-export function normalizePath(path: Path, baseUrl?: string): Path {
```

Comment:
This is a breaking change of the public API.

While `normalizePath` isn't public, it's used in the `matchRequestUrl`, which _is_ public and does support the `baseUrl` optional argument. 

I think we should revert this.